server/lib/routes/get-config.js

```diff
@@ -2,52 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-
-var config = require('../configuration');
-var clientId = config.get('oauth_client_id');
-
-var authServerUrl = config.get('fxaccount_url');
-var env = config.get('env');
-var oauthServerUrl = config.get('oauth_url');
-var profileServerUrl = config.get('profile_url');
-var allowedParentOrigins = config.get('allowed_parent_origins');
-var marketingEmailApiServerUrl = config.get('marketing_email.api_url');
-var marketingEmailPreferencesUrl = config.get('marketing_email.preferences_url');
-
 module.exports = function () {
   var route = {};
 
+  // support for this route was removed in PR #4147, config
```

Comment:
@vladikoff, @jrgm - think we should leave `/config` with all of this values around for a train to help with the crossover period after train cut? There is a tiny window of failure:

1. Old JS that requests `/config` is downloaded, request to `/config has not yet occurred.
2. content-server swap happens.
3. `/config` is requested. Oh oh, 410.